README.md

```diff
@@ -67,7 +67,7 @@ For example, any given Jaeger installation at Uber is typically processing sever
 
 ### Relationship with OpenTelemetry
```

Comment:
Hm, I recently removed this section altogether, moved it to the documentation.

https://www.jaegertracing.io/docs/next-release-v2/#relationship-with-opentelemetry